src/polyfills.ts

```diff
@@ -15,7 +15,6 @@ import 'core-js/es6/map';
 import 'core-js/es6/set';
 import 'core-js/es6/reflect';
 
-import 'core-js/es7/reflect';
```

Comment:
What was the reasoning behind removing this polyfill, or having it in the first place?